launch/launch/event_handlers/on_process_exit.py

```diff
@@ -45,15 +45,15 @@ class OnProcessExit(BaseEventHandler):
     def __init__(
         self,
         *,
-        target_action: 'ExecuteProcess' = None,
+        target_action: 'ExecuteLocal' = None,
```

Comment:
@roger-strain @mlanting meta: yeap, this definitely calls for an abstract `Execute` action.